resources/vue/src/components/site/about/AboutPage.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <div class="container">
+        <h1 class="text-center">About us</h1>
+        <p style="text-indent: 30px">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Alias laboriosam accusamus dignissimos? Soluta enim a perspiciatis corrupti cumque architecto animi excepturi incidunt? Qui cumque error inventore in iusto non sed perspiciatis veritatis quod officiis, dolorum nostrum cupiditate quisquam sequi, tenetur velit debitis! Vitae voluptates debitis autem, ad quos ullam aspernatur totam, officia necessitatibus alias quae voluptatum sunt neque, quis a repudiandae? Aut cum voluptate sit, aliquam sapiente vel! Provident corporis in tempore, placeat voluptatibus assumenda iste repellendus, ipsa asperiores eaque expedita perspiciatis sint cum pariatur distinctio eius, deserunt aliquam repellat illum labore eos quam consectetur ad! Itaque est dicta consequatur.</p>
+        <p style="text-indent: 30px">Ipsam, adipisci dolore aut et tempore omnis ad beatae sequi voluptatum quibusdam dolorum quidem amet, ducimus deserunt blanditiis unde. Ipsum ab corrupti dolore tempore rem, velit inventore, delectus iste nemo atque reiciendis quisquam ea laudantium eaque accusamus, dolores explicabo illo quidem voluptates consectetur cum. Fuga, in nobis! Sed, optio, beatae at quo ipsum impedit dolorem labore iure quaerat odit vel quis dolore eum doloribus ut enim eius officia, molestias numquam. Explicabo delectus illo officia nesciunt repellendus, est ipsa dolore iusto consectetur minima dolorum dolor illum, saepe eligendi velit ullam. Dolore quisquam unde asperiores obcaecati quam earum esse enim sapiente amet!</p>
+        <p style="text-indent: 30px">Quisquam magnam dolores perspiciatis, cupiditate, ipsa ex odio autem expedita assumenda consequatur quia amet sequi qui molestiae deserunt quod sunt nulla, perferendis quis asperiores hic animi temporibus maiores? Est quidem, quo laudantium earum eos esse commodi, harum, mollitia quibusdam illo nam amet magnam natus molestiae neque. Quos perferendis dolorem debitis, sapiente fugit deleniti doloremque delectus esse tenetur provident accusamus aut nulla cupiditate a itaque voluptas voluptate natus. Tempora ipsum excepturi dolorum hic consequatur fugit, asperiores modi consequuntur nam eum corrupti similique repellat et provident possimus culpa impedit earum? Excepturi ab natus veniam eveniet amet ad corrupti modi itaque, similique fugit?</p>
```

Comment:
Avoid styling with style as a general rule, unless you want to have something dynamic, that can't be done via CSS classes, dynamic background-image will be a good example to use style.